2 py 5.py

```diff
@@ -0,0 +1,5 @@
+# 5 – Реализуйте алгоритм перемешивания списка
+import random
+List = [1,2,3,4,5,6,7,8,9]
```

Comment:
Не лучший способ названия переменных
```suggestion
List = [1, 2, 3, 4, 5, 6, 7, 8, 9]
```